src/xdocs/config_naming.xml

```diff
@@ -160,17 +160,58 @@
        <p>
           To configure to check variables and classes identifiers,
           do not ignore variables with static modifier and allow
-          no abbreviations (enforce camel case phrase) and
-          allow no abbreviations to use (camel case phrase) and allow XML and URL abbreviations.
+          no abbreviations (enforce camel case phrase) to use
+          and allow XML and URL abbreviations.
         </p>
+        <p>Configuration:</p>
         <source>
 &lt;module name="AbbreviationAsWordInName"&gt;
   &lt;property name="tokens" value="VARIABLE_DEF,CLASS_DEF"/&gt;
   &lt;property name="ignoreStatic" value="false"/&gt;
-  &lt;property name="allowedAbbreviationLength" value="1"/&gt;
+  &lt;property name="allowedAbbreviationLength" value="0"/&gt;
   &lt;property name="allowedAbbreviations" value="XML,URL"/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code Example:</p>
+        <pre>
+public class MyClass { // OK
+  int firstNum; // OK
+  int secondNUM; // violation, it allowed only 1 consecutive capital letter
+  static int thirdNum; // OK, the static modifier would be checked
+  static int fourthNUm; // violation, the static modifier would be checked,
+                        // and only 1 consecutive capital letter is allowed
+  String firstXML; // OK, XML abbreviations are allowed
+  String firstURL; // OK, URL abbreviations are allowed
+}
+        </pre>
+        <p>
+          To configure to check variables identifiers,
+          ignore variables with static modifier and allow
+          abbreviations (up to 2 consecutive capital letters)
+          and allow CSV abbreviations.
+        </p>
+        <p>Configuration:</p>
+        <pre>
+&lt;module name="AbbreviationAsWordInName"&gt;
+  &lt;property name="tokens" value="VARIABLE_DEF"/&gt;
+  &lt;property name="ignoreStatic" value="true"/&gt;
+  &lt;property name="allowedAbbreviationLength" value="1"/&gt;
+  &lt;property name="allowedAbbreviations" value="CSV"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>Code Example:</p>
+        <pre>
+public class MyClass { // OK, ignore checking the class name
+  int firstNum; // OK
+  int secondNUm; // OK
+  int thirdNUM; // violation, it allowed only 2 consecutive
+                // capital letter
+  static int fourthNUM; // OK, variables with static modifier
+                        // would be ignored
+  String firstCSV; // OK, CSV abbreviations are allowed
+  String firstXML; // violation, XML abbreviations are not allowed
```

Comment:
abbreviations ==> abbreviation (in both cases)
as each variable is validated separately. `allowedAbbreviations` is plural to show that it is collection, not a single value.